Firestore/core/test/firebase/firestore/remote/serializer_test.cc

```diff
@@ -121,6 +134,15 @@ QueryData CreateQueryData(absl::string_view str) {
   return CreateQueryData(Query(str));
 }
 
+// Returns the full key path, including the database name, as a string.
+std::string KeyString(const std::string& key) {
```

Comment:
`KeyString` is pretty generic and doesn't indicate that it's the encoded form.

In the protocol, this is the "resource name". Maybe rename to something related to that? Alternatively, EncodedKey?

Reply:
Changed to `ResourceName`.

I deliberately avoided `Encode` in the name because in the serializer, `Encode` functions return Nanopb types, and I didn't want to disrupt that pattern. Also, since this is a helper function, the name has to be short for it to actually help with reducing verbosity. Let me know if you have other suggestions.